src/Surface.cpp

```diff
@@ -23,12 +23,27 @@ int surf_id() {
 /**
  * @brief Resets the auto-generated unique Surface ID counter to 10000.
  */
-void reset_surf_id() {
+void reset_surface_id() {
   auto_id = 10000;
```

Comment:
The reset value of 10000 actually is detailed in the doxygen comment for this method.

I do understand your proposal to define a macro constant for this value. My hesitation to do so is that 1) a `DEFAULT_CELL_INIT_ID` would only be used in one place, and 2) I'm not sure where to define a more global `DEFAULT_INIT_ID` for `Materials`, `Surfaces`, `Cells` and `Lattices`.